style.css

```diff
@@ -0,0 +1,592 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,300;1,400;1,700&family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+.bigheader {
+  color: #ec5242;
+}
+
+.salute {
+  color: #f74d1c;
```

Comment:
- [ ] @Yacoubou-seidou great work implementing the changes requested however you forgot to change the color for the q element. All elements with the orange color need to have `#ec5242` color value.